Content.Shared/Revolutionary/Components/CommandComponent.cs

```diff
@@ -0,0 +1,11 @@
+using Content.Shared.Revolutionary;
+using Robust.Shared.GameStates;
+
+namespace Content.Server.Revolutionary.Components;
+/// <summary>
+/// Given to heads at round start for Revs. Used for tracking if heads died or not.
+/// </summary>
+[RegisterComponent, NetworkedComponent, Access(typeof(SharedRevolutionarySystem))]
+public sealed class CommandComponent : Component
+{
+}
```

Comment:
This is way too generic of a component name, also it's going to leak info to cheaters so should probably actually be on server.